.env.example

```
@@ -1 +1 @@
-AV_KEY=demo
\ No newline at end of file
+API_KEY=demo
```

Can we revert this API key rename? I don't want to break env configs 
```suggestion
AV_KEY=demo
```